build/ghost-buster/Dockerfile

```diff
@@ -1,33 +1,35 @@
-FROM alpine:3.4
+FROM alpine:3.6
```

Comment:
First I’ve upgraded alpine to 3.6. In this release nodejs-lts is right v6, which is required for ghost v1.